python-sdk/tests_integration/astro_deploy/astro_sdk_connections.py

```diff
@@ -0,0 +1,87 @@
+import os
+from datetime import datetime
+from textwrap import dedent
+
+import yaml
+from airflow.decorators import dag, task
+from airflow.models import Connection
+from airflow.utils.db import create_default_connections
+from airflow.utils.session import create_session
+
+connections_yaml = f"{os.environ['AIRFLOW_HOME']}/include/connections_template.yaml"
```

Comment:
@vatsrahul1001 How are we getting this file?

Reply:
Removed this change, will be using secrets backend for connection storage